test/test_functionalization.py

```diff
@@ -475,6 +475,21 @@ def forward(self, arg0_1):
     as_strided_copy_1 = torch.ops.aten.as_strided_copy.default(as_strided_scatter, [2], [2], 1)
     copy_ = torch.ops.aten.copy_.default(arg0_1, as_strided_scatter);  arg0_1 = None
     return as_strided_scatter
+    """)
+
+        # NB: even with reapply_views=True, we expect to see scatter op
+        reinplaced_logs = self.get_logs(f, torch.ones(2, 2), reapply_views=True, run_reinplace=True)
```

Comment:
you can just use `run_reinplace=False` in these tests - I wrote a reinplacing pass a while ago in FX that I tested here, but it didn't end up getting used (inductor has its own reinplacing pass).

These expecttests are useful, but oh man they take up a lot of space lol

Reply:
> These expecttests are useful, but oh man they take up a lot of space lol

I know right :( my PR looks massive